src/app/modules/dashboard/pages/dashboard/components/cpu-stats/cpu-stats.component.spec.ts

```diff
@@ -0,0 +1,35 @@
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+import {RouterTestingModule} from '@angular/router/testing';
+import {HttpClientTestingModule} from '@angular/common/http/testing';
+import {NoopAnimationsModule} from '@angular/platform-browser/animations';
+import {ToastrModule} from 'ngx-toastr';
+
+import {CpuStatsComponent} from './cpu-stats.component';
+import {MaterialModule} from '../../../../../../shared/material.module';
+
+describe('CpuStatsComponent', () => {
+    let component: CpuStatsComponent;
+    let fixture: ComponentFixture<CpuStatsComponent>;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            imports: [
+                NoopAnimationsModule,
+                MaterialModule,
+                HttpClientTestingModule,
+                RouterTestingModule,
+                ToastrModule.forRoot()
```

Comment:
Missing `ChartsModule` import which fails the test.
```suggestion
                ToastrModule.forRoot(),
                ChartsModule
```